packages/compass-workspaces/src/components/workspaces.tsx

```diff
@@ -41,6 +41,9 @@ import {
   useTabConnectionTheme,
   useConnectionRepository,
 } from '@mongodb-js/compass-connections/provider';
+import { usePreference } from 'compass-preferences-model/provider';
+
+type Tooltip = [string, string][];
```

Comment:
What are the two things? type and text? Maybe for type it can be narrower than string?

Reply:
Oh nevermind I see now it is just text.